README.md

```diff
@@ -27,5 +27,14 @@ Now it is time to create the kernel on which your notebook will run, open the te
 python -m ipykernel install --user --name myenv
 ```
 Now just select this kernel you created when running the NotebookAnalysisResults notebook.
+## Run the web API
+Use the terminal or an Anaconda Prompt for the following steps:
+1. Move to the folder containing the main.py fastAPI module
```

Comment:
```suggestion
1. Move to the folder containing the `main.py` module
```

`main.py` is actually a pynblint module, not a FastAPI module. In that script we use FastAPI